include/mapbox/geojsonvt/tile.hpp

```diff
@@ -17,12 +17,17 @@ namespace detail {
 
 class InternalTile {
 public:
+    const uint16_t extent;
+    bool is_solid = false;
     const uint8_t z;
     const uint32_t x;
     const uint32_t y;
 
+    const double z2;
```

Comment:
You can still mark them as `private` even if they're in a different location. Code style-wise, we typically list all member functions first, rather than interspersing them.

Reply:
In this case it doesn't matter much because it's an internal type — I think we can leave it like this.